packages/plugins/documentation/admin/src/pages/utils/useReactQuery.js

```diff
@@ -1,14 +1,28 @@
 import { useQuery, useMutation, useQueryClient } from 'react-query';
-import { useNotification } from '@strapi/helper-plugin';
-import { fetchDocumentationVersions, deleteDoc, regenerateDoc, updateSettings } from './api';
+import { useNotification, useFetchClient } from '@strapi/helper-plugin';
+import pluginId from '../../pluginId';
 import getTrad from '../../utils/getTrad';
 
 const useReactQuery = () => {
   const queryClient = useQueryClient();
   const toggleNotification = useNotification();
-  const { isLoading, data } = useQuery('get-documentation', () =>
-    fetchDocumentationVersions(toggleNotification)
-  );
+  const { isLoading, data } = useQuery('get-documentation', async () => {
```

Comment:
```suggestion
  const { isLoading, data } = useQuery(['get-documentation', pluginId], async () => {
```

Dependencies should always be listed in the query-key